<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Oz Single Cells Challenge 2019</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cerulean.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Oz Single Cells Challenge 2019</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Oz Single Cells Challenge 2019</h1>

</div>


<div id="introduction-to-computational-challenge" class="section level1">
<h1>Introduction to Computational Challenge</h1>
<p>As part of Oz Single Cell 2019 conference, we are hosting a single cell data analysis challenge. The challenge aims to foster the development of ideas and approaches to better utilise data arising from single cell sequencing technology, specifically using one or more of three single cell data sets. We encourage innovation from any aspect of single cell analytics from the extraction of biological insight through to complex analytical approaches. Possible entries to the challenge include:</p>
<ul>
<li>Technological innovation including better quality control, normalization pipeline, frameworks for comparison of pipelines,</li>
<li>Biological discovery or confirmation,</li>
<li>Analytical innovation including integration with other data sets, high-level visualisation and others. Please note that entries are not limited to the above suggestions and participants are strongly encouraged to come up with new ways for utilizing these types of data.</li>
</ul>
</div>
<div id="challenge-entrants" class="section level1">
<h1>Challenge Entrants</h1>
<p>Teams can consist of any number of people, but we encourage teams to have a mix of students, postdocs, and more senior academic staff. To register your team, please fill in our form <a href="https://docs.google.com/forms/d/e/1FAIpQLSebsq7sjQnaodGPNx0vY_KFnsgC2FdvTm-ODvYEt2E-TbiXWg/viewform?usp=sf_link">here</a>, note that this is not binding and you can change your mind later. On 15th July, we will ask each team entering the challenge to provide us with a 200-word description of their entry.</p>
</div>
<div id="judging-criteria" class="section level1">
<h1>Judging criteria</h1>
<p>Participants are invited to present their analysis during the Oz Single Cell conference. Each team will be allocated a 3 min rapid-fire talk and a panel will judge the entries. The entries will be judged according to the novelty and potential impact on future single cell work. This could be in any of the following areas:</p>
<ul>
<li>Computational techniques</li>
<li>Analysis methodology</li>
<li>Biological insight</li>
</ul>
<p>In addition, the clarity and coherence of the presentation will be taken into account.</p>
</div>
<div id="datasets" class="section level1">
<h1>Datasets</h1>
<div id="dataset-1---swarbrick-lab-cite-seq-data" class="section level3">
<h3>Dataset 1 - Swarbrick Lab CITE-seq data</h3>
<div id="short-summary" class="section level5">
<h5>SHORT SUMMARY</h5>
<p>Dataset from the Swarbrick lab provides the participants with an opportunity to explore CITE-seq, a powerful new technology that allows simultaneous characterisation of both RNA and surface proteins. Hackers are invited to build stories around the biological (metastatic lymph node) or technical aspects (normalisation, visualisation, integration) of the data.</p>
</div>
<div id="background" class="section level5">
<h5>BACKGROUND</h5>
<p>Cell-surface proteins were historically crucial in describing cellular states, and were often a more reliable indicator of cellular activity and function than RNA expression. The Swarbrick Lab from Garvan Institute has implemented CITE-seq technology (Stoeckius et al. 2017) that couples protein and RNA information for individual cells, allowing a more powerful characterization of cellular phenotypes than transcriptome measurements alone.</p>
</div>
<div id="data" class="section level5">
<h5>DATA</h5>
<p>In the experiment below we applied multimodal analysis to phenotype immune cells in a human metastatic lymph node. After spiking in mouse cells (3T3) and staining with 96 CITE-seq antibodies (with IgG1, IgG2a and IgG2b as isotype controls), the samples were analysed on Chromium 10x. All of the above are available in raw format or normalized and scaled Seurat objects (v3).</p>
</div>
<div id="challenges" class="section level5">
<h5>CHALLENGES</h5>
<ol style="list-style-type: decimal">
<li>Subset and visualize different cell populations using integrated RNA and CITE-seq data to determine differences between cellular populations</li>
<li>Improve the default statistical framework for the QC of CITE antibodies using mouse spike-ins and IgG controls</li>
<li>Compare CITE profiles with publicly available cell-type signatures (i.e. www.immgen.org)</li>
</ol>
</div>
<div id="download" class="section level5">
<h5>DOWNLOAD</h5>
<ol style="list-style-type: decimal">
<li>RAW CITE-seq COUNTS (csv file: <a href="http://www.maths.usyd.edu.au/u/SMS/bioinformatics/wwwnb/CITE-seq/Swarbrick_metastatic_lymphnode.CITE.csv">link</a>)</li>
</ol>
<ul>
<li>Swarbrick_metastatic_lymphnode.CITE.csv</li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>SEURAT OBJECT (.RData file: <a href="http://www.maths.usyd.edu.au/u/SMS/bioinformatics/wwwnb/CITE-seq/Swarbrick_metastatic_lymphnode.seurat.Rdata">link</a>)</li>
</ol>
<ul>
<li>Swarbrick_metastatic_lymphnode.seurat.Rdata</li>
<li>Seurat (v3) object with raw and processed data human and mouse data, with CITE-seq counts (RNA and ADT (antibody-derived tags)) included.</li>
<li>data processed with Seurat_3.0.0.9100, normalized with SCTransform (stored in SCT assay)</li>
</ul>
<ol start="3" style="list-style-type: decimal">
<li>ANTIBODY ANNOTATION (csv file: <a href="http://www.maths.usyd.edu.au/u/SMS/bioinformatics/wwwnb/CITE-seq/antibody_isotype.csv">link</a>)</li>
</ol>
<ul>
<li>antibody_isotypes.csv</li>
</ul>
</div>
</div>
<div id="dataset-2---benchmarking-datasets" class="section level3">
<h3>Dataset 2 - Benchmarking datasets</h3>
<div id="dataset-2.1---ding-et-al.-benchmark-datasets" class="section level4">
<h4>Dataset 2.1 - Ding et al. Benchmark Datasets</h4>
<div id="description" class="section level5">
<h5>DESCRIPTION</h5>
<p>A multitude of single-cell RNA sequencing methods have been developed in recent years, with dramatic advances in scale and power, and enabling major discoveries and large scale cell mapping efforts. However, these methods have not been systematically and comprehensively benchmarked. Here, we directly compare seven methods for single cell and/or single nucleus profiling from three types of samples – cell lines, peripheral blood mononuclear cells and brain tissue – generating 36 libraries in six separate experiments in a single center. To analyze these datasets, we developed and applied scumi, a flexible computational pipeline that can be used for any scRNA-seq method. We evaluated the methods for both basic performance and for their ability to recover known biological information in the samples. Our study will help guide experiments with the methods in this study as well as serve as a benchmark for future studies and for computational algorithm development.</p>
<div class="figure">
<img src="figures/DingEtAl.png" alt="Experimental design for Ding et al. benchmark data. (Ding et al, 2019)" />
<p class="caption">Experimental design for Ding et al. benchmark data. (Ding et al, 2019)</p>
</div>
</div>
<div id="download-1" class="section level5">
<h5>DOWNLOAD</h5>
<p><strong>PBMC data</strong></p>
<p>There are two experiments (PBMC1 and PBMC2), each with x7 scRNA-seq methods (Sci-Seq, Smart-seq2, CEL-seq2, 10x_v2,Seq-well, Drop-seq, inDrops): 31021 cells:</p>
<p><a href="https://portals.broadinstitute.org/single_cell/study/SCP424/single-cell-comparison-pbmc-data" class="uri">https://portals.broadinstitute.org/single_cell/study/SCP424/single-cell-comparison-pbmc-data</a></p>
<p><strong>Cortex data</strong></p>
<p>Cortex data 13783 cells, the mouse Cortex single nucleus RNA-seq (snRNA-seq) experiments. These consist of two experiments (Cortex1 and Cortex2), each with 4 snRNA-seq methods (10X, DroNc-Seq, Sci-RNA-Seq, and Smart-seq2):</p>
<p><a href="https://portals.broadinstitute.org/single_cell/study/SCP425/single-cell-comparison-cortex-data" class="uri">https://portals.broadinstitute.org/single_cell/study/SCP425/single-cell-comparison-cortex-data</a></p>
<p><strong>Cell mixture</strong></p>
<p>cells mixture (HEK293 and NIH3T3) experiments: 28580 cells. These consist of two experiments (Mixture1 and Mixture2), each with x7 scRNA-seq methods. This dataset allowed us to better understand the ability of different methods to detect gene expression (detecting doublets):</p>
<p><a href="https://portals.broadinstitute.org/single_cell/study/SCP424/single-cell-comparison-pbmc-data" class="uri">https://portals.broadinstitute.org/single_cell/study/SCP424/single-cell-comparison-pbmc-data</a></p>
</div>
<div id="reference" class="section level5">
<h5>REFERENCE</h5>
<p>Ding, J. et al., 2019. Systematic comparative analysis of single cell RNA-sequencing methods. bioRxiv, p.632216. Available at: <a href="https://www.biorxiv.org/content/10.1101/632216v2" class="uri">https://www.biorxiv.org/content/10.1101/632216v2</a>.</p>
</div>
</div>
<div id="dataset-2.2---cellbench-single-cell-rna-seq-benchmarking" class="section level4">
<h4>Dataset 2.2 - CellBench: single cell RNA-seq benchmarking</h4>
<div id="description-1" class="section level5">
<h5>DESCRIPTION</h5>
<p>CellBench uses three human lung adenocarcinoma cell lines HCC827, H1975 and H2228, which were cultured separately, and then processed in three different ways. Firstly, single cells from each cell line were mixed in equal proportions, with libraries generated using three different protocols: CEL-seq2, Drop-seq (with Dolomite equipment) and 10X Chromium. Secondly, the single cells were sorted from the three cell lines into 384-well plates, with an equal number of cells per well in different combinations (generally 9-cells, but with some 90-cell population controls). Thirdly, RNA was extracted in bulk for each cell line and the RNA was mixed in 7 different proportions and diluted to single cell equivalent amounts ranging from 3.75pg to 30pg and processed using CEL-seq 2 and SORT-seq. ERCC spike-in controls were present in samples processed using the 2 plate-based technologies (CEL-seq2 and SORT-seq).</p>
<div class="figure">
<img src="figures/cellBench_expr_design.png" alt="Experimental design for cell bench data. (Tian et al, 2019)" />
<p class="caption">Experimental design for cell bench data. (Tian et al, 2019)</p>
</div>
</div>
<div id="download-2" class="section level5">
<h5>DOWNLOAD</h5>
<p>Raw data from this series of experiments is available under GEO accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE118767">GSE118767</a>. The processed count data obtained from <code>scPipe</code> and stored in R objects that use the <code>SingleCellExperiment</code> class can be downloaded from <a href="https://github.com/LuyiTian/CellBench_data" class="uri">https://github.com/LuyiTian/CellBench_data</a>. The website contains instructions for getting the count data and metadata (including annotations) for each dataset. All data is post sample quality control, without gene filtering.</p>
</div>
<div id="reference-1" class="section level5">
<h5>REFERENCE</h5>
<p>Tian, L. et al., 2019. Benchmarking single cell RNA-sequencing analysis pipelines using mixture control experiments. Nature Methods, 16(6), pp.479–487. Available at: <a href="https://doi.org/10.1038/s41592-019-0425-8" class="uri">https://doi.org/10.1038/s41592-019-0425-8</a>.</p>
<!-- ### Dataset 3 - the Hackathon -->
<!-- ##### Description: -->
<!-- Spatial transcriptomics data of prostate cancer tissues were generated by Lunderberg’s Lab. The study has been published in Nature Communications (Berglund et al, 2018). Prior to the workshop, participants are encouraged to read the paper and to attend the two talks on spatial transcriptomics in the morning technology session. -->
<!-- ![Study design for spatial transcriptomics (ST) in prostate cancer. (Berglund et al, 2018)](figures/Berglund_figure.png) -->
<!-- ##### Download: -->
<!-- ##### Reference: -->
<!-- Berglund, E. et al., 2018. Spatial maps of prostate cancer transcriptomes reveal an unexplored landscape of heterogeneity. Nature Communications, 9(1), p.2419. Available at: https://doi.org/10.1038/s41467-018-04724-5. -->
</div>
</div>
</div>
</div>
<div id="contact" class="section level1">
<h1>Contact</h1>
<p>If you have any questions about this challenge, feel free to email us at: <a href="mailto:singlecellchallenge@maths.usyd.edu.au">singlecellchallenge@maths.usyd.edu.au</a></p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
