<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Introduction to Advanced computational workshop - Hackathon</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cerulean.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Oz Single Cells Challenge 2019</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Challenge</a>
</li>
<li>
  <a href="workshop.html">Workshop</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Introduction to Advanced computational workshop - Hackathon</h1>

</div>


<div id="aim" class="section level1">
<h1>Aim</h1>
<p>To facilitate collaboration among cellular genomics researchers around Australia and to explore and solve challenges in analysing one of the next frontier technologies, namely spatial transcriptomics (ST). ST data contains both imaging pixel data and sequencing read data from the same tissue section, thus creating opportunities for new analysis approaches, which will be discussed at the Hackathon.</p>
</div>
<div id="the-workshop-format" class="section level1">
<h1>The workshop format</h1>
<p>At the workshop, participants will form Hackathon teams to encourage team diversity regarding geographical locations, organisations, genders, and expertise. The Hackathon will mainly focus on interactive discussion and idea generation rather than real-time execution of data analysis programs. Attendants can produce algorithms and pseudo-codes to demonstrate how to implement the ideas, but no intensive computational runs are required. The teams should use their own laptops, and in rare cases, use VPN connection to access to HPC.</p>
<p>Prior to the workshop, participants are encouraged to attend the two technology talks on spatial transcriptomics in the morning session Sunday 21st July. For those interested in working with the data before the workshop, the data can be downloaded as instructed below. At the end of the workshop, each team will deliver a flash presentation on their analysis ideas on spatial transcriptomics (3 minutes/team). There will be prizes for the winning teams.</p>
</div>
<div id="hackathon-dataset" class="section level1">
<h1>Hackathon Dataset</h1>
<div id="dataset-description" class="section level4">
<h4>DATASET DESCRIPTION</h4>
<p>Spatial transcriptomics data of prostate cancer tissues were generated by Lunderberg’s Lab, published in Nature Communications (Berglund et al, 2018). The dataset contains Slide-seq sequencing data for 12 tissue sections. Raw and processed count matrices and high-resolution images can be downloaded below. Raw Fastq files are available at European Genome–Phenome Archive (EGA) under the accession number EGAS0000100300.</p>
<div class="figure">
<img src="figures/Berglund_figure.png" alt="Study design for spatial transcriptomics (ST) in prostate cancer. (Berglund et al, 2018)" />
<p class="caption">Study design for spatial transcriptomics (ST) in prostate cancer. (Berglund et al, 2018)</p>
</div>
</div>
<div id="files" class="section level4">
<h4>FILES</h4>
<ol style="list-style-type: decimal">
<li>Raw count matrices: 12 tsv files containing raw read count data of Slide-seq spots in 12 tissue sections. (raw_matrices.zip <a href="http://www.maths.usyd.edu.au/u/SMS/bioinformatics/wwwnb/ProstateCancer/raw_matrices.zip">link</a>)</li>
<li>Coordinate-adjusted count matrices: we generated 12 tsv files, containing cooordinate-adjusted count data that match H&amp;E images (adjusted_matrices.zip <a href="http://www.maths.usyd.edu.au/u/SMS/bioinformatics/wwwnb/ProstateCancer/adjusted_matrices.zip">link</a>)</li>
<li>High-resolution H&amp;E images: 12 raw images kindly provided by the authors (images.zip <a href="http://www.maths.usyd.edu.au/u/SMS/bioinformatics/wwwnb/ProstateCancer/images.zip">link</a>)</li>
</ol>
</div>
<div id="suggested-problems" class="section level4">
<h4>SUGGESTED PROBLEMS</h4>
<ul>
<li>QC and normalisation of imaging data and sequencing data within a tissue section and between tissue sections</li>
<li>Identification of cell types and microenvironments</li>
<li>Identification of cell-cell interactions</li>
<li>Integration of imaging data and sequencing read count data</li>
</ul>
</div>
</div>
<div id="reference" class="section level1">
<h1>Reference:</h1>
<p>Berglund, E. et al., 2018. Spatial maps of prostate cancer transcriptomes reveal an unexplored landscape of heterogeneity. Nature Communications, 9(1), p.2419. Available at: <a href="https://doi.org/10.1038/s41467-018-04724-5" class="uri">https://doi.org/10.1038/s41467-018-04724-5</a>.</p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
